package.json

```diff
@@ -1,15 +1,18 @@
 {
   "name": "teamsfx",
   "scripts": {
-    "setup": "npm install && npm run bootstrap --hoist && npm run install-eslint",
+    "setup": "npm install && npm run build",
     "setup-failpoint": "npm run setup && npx lerna run build-failpoint",
```

Comment:
`setup` shouldn't exist anymore, as above

Reply:
Thanks.
if build set to postinstall, install do the whole setup jobs.
besides, I think our teams used to use this setup script, so keep it.